### PR TITLE
Split URI to user-friendly configuration parameters

### PR DESCRIPTION
**Pull request by kirill-levitskiy**
Currently one have to enter an MSSQL URI as a string like:

mssql://username:password@localhost/database

Issues with it:

Not very user friendly
Password is visible in UI and not protected
However, advantages are that many possible settings could be build in as encoded in URI. We need t good solution to combine streights but not weaknesses.

Issue: [https://github.com/elasticio/mssql-component/issues/6](url)

### CODE REVIEW

File: lib/actions/select.js [hunk 1]
```diff
@@ -14,7 +14,18 @@ let connection;
  * @param cfg
  */
 function init(cfg) {
-    const conString = cfg.uri;
+    const conString = 'mssql://'
+      + cfg.username
```
Comment (zubairov) [inline lib/actions/select.js]: Username can't contain ':' right?

File: lib/actions/select.js [hunk 2]
```diff
@@ -14,7 +14,18 @@ let connection;
...
+      + ':'
+      + cfg.password
```
Comment (zubairov) [inline lib/actions/select.js]: same here, no ':' and '@' allowed right?

File: package.json [hunk 3]
```diff
@@ -25,27 +25,27 @@
...
     "co": "^4.6.0",
     "co-mssql": "^1.3.0",
-    "elasticio-node": "0.0.8",
-    "elasticio-sailor-nodejs": "2.2.0",
-    "mssql": "^3.3.0",
-    "request": "^2.75.0",
-    "request-promise": "^4.1.1"
+    "elasticio-node": "0.0.9",
+    "elasticio-sailor-nodejs": "^2.2.1",
+    "mssql": "^4.1.0",
```
Comment (zubairov) [inline package.json:30]: Good idea 👍

File: lib/actions/select.js (hunk 1 above)
Comment (kirill-levitskiy) [reply lib/actions/select.js]: The issue was fixed in the last commit by escaping special characters in username

File: lib/actions/select.js (hunk 2 above)
Comment (kirill-levitskiy) [reply lib/actions/select.js]: The issue was fixed in the last commit by escaping special characters in password

**Pull request merged**